forte/utils/utils.py

```diff
@@ -78,8 +79,10 @@ def get_class_name(o, lower: bool = False) -> str:
         return o.__name__
 
 
-def get_class(full_class_name: str, module_paths: Optional[List[str]] = None):
-    r"""Returns the class based on class name.
+def get_class_nc(
+    full_class_name: str, module_paths: Optional[List[str]] = None
+):
+    r"""Returns the class based on class name, not cached.
```

Comment:
please explain what is "not cached" to make this documentation clear.

Reply:
This is now fixed in the new fe-factored code that combine the 2 methods into one with optional parameters